BrQTitleCard.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <q-card
+    class="text-center"
+    inline
+    color="white">
+    <h6
+      class="q-my-none text-subtitle1 text-center q-py-sm q-px-md">
+      {{title}}
+      <slot name="head" />
+    </h6>
+    <q-space />
+    <slot name="icon" />
+    <q-separator />
+    <q-card-section class="q-pa-none">
+      <slot name="body" />
+    </q-card-section>
+  </q-card>
+</template>
+
+<script>
+/*!
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
```

Comment:
Shouldn't all copyright year be the same (i.e 2020), since it is the initial version?

Reply:
Some of the code here was copied from other projects which existed earlier.  For new code we should start with 2020.  @JoshDunnDev perhaps you know which code is new vs old?